test/e2e/framework/pv/.import-restrictions

```diff
@@ -1,9 +1,23 @@
 # This E2E framework sub-package is currently allowed to use arbitrary
-# dependencies, therefore we need to override the restrictions from
-# the parent .import-restrictions file.
+# dependencies except of k/k/pkg, therefore we need to override the
+# restrictions from the parent .import-restrictions file.
 #
 # At some point it may become useful to also check this package's
 # dependencies more careful.
 rules:
+  - selectorRegexp: "^k8s[.]io/kubernetes/pkg"
+    allowedPrefixes:
+    - k8s.io/kubernetes/pkg/api/v1/pod
+    - k8s.io/kubernetes/pkg/api/legacyscheme
+    - k8s.io/kubernetes/pkg/apis/
+    - k8s.io/kubernetes/pkg/features
+    - k8s.io/kubernetes/pkg/kubelet/server/metrics
+    - k8s.io/kubernetes/pkg/proxy/util
+    - k8s.io/kubernetes/pkg/securitycontext
+    - k8s.io/kubernetes/pkg/util/parsers
+    - k8s.io/kubernetes/pkg/util/slice
+    - k8s.io/kubernetes/pkg/volume
```

Comment:
Let's not choose specific allowed packages for now, i.e. leave this file as it was before.

Reply:
There is only one direct dependency here: k8s.io/kubernetes/pkg/api/v1/pod, the rest is indirect.
Would it make sense to add a comment to separate indirect deps instead of allowing to add anything?

Reply:
done